python/simple/v1/ampli/ampli.py

```diff
@@ -508,7 +508,17 @@ def load(self, options: Optional[LoadOptions] = None):
         
         if not self.client.configuration.plan:
             self.client.configuration.plan = DEFAULT_CONFIGURATION.plan
-    
+
+        # Python SDK min version 1.1.0 supports IngestionMetadata
+        # This lazy module loading is for backward compatible concern.
+        import amplitude
+        if hasattr(amplitude, "IngestionMetadata"):
```

Comment:
This makes sense to me. It's easier than the plugin ways.

Reply:
Can you re format this like 
```
try:
    from amplitude import IngestionMetadata
except ImportError:
    IngestionMetadata = None

...
if IngestionMetadata:
    ....
```

Reply:
The import statement could go to top of the file

Reply:
I try to put the import in the function, to scope it, not pollute the global namespace. If we want to use `try`, think this might be better:
```python
try:
    from amplitude import IngestionMetadata
    self.client.configuration.ingestion_metadata = IngestionMetadata(
        source_name="python-python-ampli",
        source_version="1.0.0"
    )
except ImportError:
    pass
```
I don't have a preference on using `getattr(amplitude, "IngestionMetadata")` or `from amplitude import IngestionMetadata`, or even `__import__` or `importlib.import_module( )`. Let me know if you have any preference and reason, I can try that approach. Thanks!

Reply:
Nice @liuyang1520 I like it

Reply:
If we don't want to expose the IngestionMetadata class just use it in this method. I'm ok with either way.